src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -643,16 +645,26 @@ class Diagram extends Component<Props> {
     return target && target.model instanceof RouterNodeModel
   }
 
+  handleTabChanged = (tab: string) => {
+    this.setState({ currentTab: tab })
+  }
+
   render() {
     return (
       <MainContent.Wrapper>
-        <WorkflowToolbar />
+        <WorkflowToolbar tabChange={this.handleTabChanged} />
+        {this.state.currentTab === 'variables' && <VariablesEditor />}
         <Fragment>
           <div
             id="diagramContainer"
             ref={ref => (this.diagramContainer = ref)}
             tabIndex={1}
-            style={{ outline: 'none', width: '100%', height: '100%' }}
+            style={{
```

Comment:
Can you add a class to this div and move all these styles to it and add a conditional class with the `classnames` library for the display none

Reply:
@samuelmasse please do. Not because it's someone else's job to apply style properly that you want to leave artefacts like this.

You can if you leave a comment in your PR and in the file that this is only for prototyping purposes but I wouldn't abuse of this